ebuild-writing/variables/text.xml

```diff
@@ -486,6 +486,36 @@ infrastructure</uri>; in that case, the benefit to using third-party mirror
 list does not outweigh the burden of maintaining it.
 </p>
 
+</body>
+</subsection>
+
+
+<subsection>
+<title>Lifting restrictions</title>
+<body>
+
+<p>
+In EAPI 8, individual items in <c>SRC_URI</c> can be exempted from automatic
+mirroring and fetching restrictions (imposed by <c>RESTRICT="mirror"</c> and
+<c>RESTRICT="fetch"</c>) by prefixing the addresses with <c>mirror+</c> or
+<c>fetch+</c>.  For example, in the following ebuild,
+</p>
+
+<codesample lang="ebuild">
+EAPI="8"
+
+SRC_URI="${P}.tar.gz
+	mirror+https://dev.gentoo.org/~larry/distfiles/${P}-patches.tar.gz"
```

Comment:
It is only an example, but I still think that `${P}-patches.tar.gz` is misleading. There are few (if any) fetch-restricted packages that can be patched.

Maybe use `${P}-addons` or `${P}-config` instead?